include/pcmki/pcmki_transition.h

```diff
@@ -28,39 +28,53 @@ typedef enum {
 typedef struct te_timer_s crm_action_timer_t;
 typedef struct crm_graph_s crm_graph_t;
 
-enum pcmki__synapse_flags {
-    pcmki__synapse_ready       = (1 << 0),
-    pcmki__synapse_failed      = (1 << 1),
-    pcmki__synapse_executed    = (1 << 2),
-    pcmki__synapse_confirmed   = (1 << 3),
+enum pcmk__synapse_flags {
+    pcmk__synapse_ready       = (1 << 0),
+    pcmk__synapse_failed      = (1 << 1),
+    pcmk__synapse_executed    = (1 << 2),
+    pcmk__synapse_confirmed   = (1 << 3),
 };
 
 typedef struct synapse_s {
     int id;
     int priority;
 
-    uint32_t flags; // Group of pcmki__synapse_flags
+    uint32_t flags; // Group of pcmk__synapse_flags
 
     GList *actions;           /* crm_action_t* */
     GList *inputs;            /* crm_action_t* */
 } synapse_t;
 
-const char *pcmk__synapse_name(synapse_t *c);
+const char *synapse_state_str(synapse_t *synapse);
 
 #define pcmk__set_synapse_flags(synapse, flags_to_set) do {             \
         (synapse)->flags = pcmk__set_flags_as(__func__, __LINE__,       \
             LOG_TRACE,                                                  \
-            "Synapse", pcmk__synapse_name(synapse),                       \
+            "Synapse", synapse_state_str(synapse),                       \
             (synapse)->flags, (flags_to_set), #flags_to_set);           \
     } while (0)
 
 #define pcmk__clear_synapse_flags(synapse, flags_to_clear) do {         \
         (synapse)->flags = pcmk__clear_flags_as(__func__, __LINE__,     \
             LOG_TRACE,                                                  \
-            "Synapse", pcmk__synapse_name(synapse),                       \
+            "Synapse", synapse_state_str(synapse),                       \
             (synapse)->flags, (flags_to_clear), #flags_to_clear);       \
     } while (0)
 
+#define pcmk__set_crm_action_flags(crm_action, flags_to_set) do {       \
+        (crm_action)->flags = pcmk__set_flags_as(__func__, __LINE__,    \
+            LOG_TRACE,                                                  \
+            "CRM Action", pcmk__crm_action_name(synapse),               \
+            (crm_action)->flags, (flags_to_set), #flags_to_set);        \
+    } while (0)
+
+#define pcmk__clear_crm_action_flags(synapse, flags_to_clear) do {         \
+        (crm_action)->flags = pcmk__set_flags_as(__func__, __LINE__,    \
+            LOG_TRACE,                                                  \
+            "CRM Action", pcmk__crm_action_name(synapse),               \
+            (crm_action)->flags, (flags_to_clear), #flags_to_clear);    \
+    } while (0)
+
```

Comment:
I'm guessing these are supposed to go with the next PR?

BTW I eventually plan to rename crm_action_t to pcmk__graph_action_t, so ...graph_action... might be good here

Reply:
Yup, I took them out and am putting them in the next PR :)